### PR TITLE
Add indexes to tables

### LINKED ISSUE #382: Create indexes in postgres
Currently there's no indexes, which means a lot of sequential scans. Here's the indexes I created:

```sql
create index _t2b_emoji_id on emoji(emoji_id);
create index _t2b_emoji_mxc on emoji(mxc_url);
create index _t2b_event_store_mxid on event_store(matrix_id);
create index _t2b_event_store_discordid on event_store(discord_id);
create index _t2b_discord_msg_store_msgid on discord_msg_store(msg_id);
```

and table row counts at the time:

|table|rows|
|-----|------|
|`emoji`|4314|
|`event_store`|8701140|
|`discord_msg_store`|7618470|
|`user_id_discord_id`|0|
|`discord_id_token`|0|
|`dm_rooms`|0|

### PR DESCRIPTION
**Pull request by Half-Shot**
Fixes #382

### CODE REVIEW

File: src/db/schema/v10.ts [hunk 1]
```diff
@@ -0,0 +1,54 @@
+/*
+Copyright 2018 matrix-appservice-discord
```
Comment (Sorunome) [inline src/db/schema/v10.ts]: it's 2019 xP

File: src/db/schema/v10.ts [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+        idx_room_entries_id: ["room_entries", "id"],
+        idx_room_entries_matrix_id: ["room_entries", "matrix_id"],
+        idx_room_entries_remote_id: ["room_entries", "remote_id"],
+    };
+
+    public async run(store: DiscordStore): Promise<void> {
+        try {
+            await Promise.all(Object.keys(this.INDEXES).map(async (indexId: string) => {
+                const ids = this.INDEXES[indexId];
+                return store.db.Exec(`CREATE INDEX ${indexId} ON ${ids[0]}(${ids[1]})`);
```
Comment (Sorunome) [inline src/db/schema/v10.ts]: what if we have multiple combinations to index? we don't have that, do we?

File: src/store.ts [hunk 3]
```diff
@@ -29,7 +27,9 @@ import { DbUserStore } from "./db/userstore";
 import {
     RoomStore, UserStore,
 } from "matrix-appservice-bridge";
+
 const log = new Log("DiscordStore");
+export const CURRENT_SCHEMA = 10;
```
Comment (Sorunome) [inline src/store.ts]: why the location switch?

File: src/db/schema/v10.ts (hunk 1 above)
Comment (Half-Shot) [reply src/db/schema/v10.ts]: BAH HUMBUG

File: src/db/schema/v10.ts (hunk 2 above)
Comment (Half-Shot) [reply src/db/schema/v10.ts]: Nope. If we ever do, then we can just improve upon this in the next schema

File: src/store.ts (hunk 3 above)
Comment (Half-Shot) [reply src/store.ts]: I was annoyed that we had the schema in the middle of our imports

**Comment by Half-Shot** [comment]
>did you test with both sqlite and postgres?

The syntax is the same but...no I didn't. I'll do that

**Pull request merged**